### PR TITLE
feat(ui-search): Add integrations to search (APP-707)

### PR DESCRIPTION
**Pull request by billyvg**
Adds integrations and legacy integrations into search

image

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/organization/navigationConfiguration.jsx [hunk 1]
```diff
@@ -67,7 +67,9 @@ const organizationNavigation = [
       {
         path: `${pathPrefix}/integrations/`,
         title: t('Integrations'),
-        description: t('Manage integrations for an organization'),
+        description: t(
+          'Manage organization-level integrations, including: Slack, Github, Bitbucket, Jira, and Azure DevOps'
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/settings/organization/navigationConfiguration.jsx]:
```suggestion
          'Manage organization-level integrations, including: Slack, GitHub, Bitbucket, Jira, and Azure DevOps'
```

**Comment by eurica** [comment]
UI Looks good. Even having 2 Slacks is the right experience, since the legacy Slack is how you connect to some other chat apps like [Discord](https://github.com/getsentry/sentry-plugins/issues/249#issuecomment-357247457)

**Review by evanpurkhiser — APPROVED** [review]

**Comment by guitcastro** [comment]
I am not finding the slack legacy app in search, how can I connect to Discord now?

**Comment by arjunbazinga** [comment]
Same as @guitcastro , bumping issue

**Comment by marcdubs** [comment]
Same here. I don't think it's possible to use Discord now without that slack legacy integration.

**Comment by arjunbazinga** [comment]
@marcdubs there was a workaround posted here:  https://github.com/getsentry/sentry/issues/10925#issuecomment-631227071

**Pull request merged**